### DISCUSSION TITLE
Mastodon account for this project

### DISCUSSION THREAD

**Discussion by linux-lukas** [joseantmazonsb/linguard#54] [General]
Would a Mastodon account for this project be possible?

**Comment by joseantmazonsb** [comment]
i don't really know anything about mastodon, any reasons why it would be a good thing to open an account for the project?

**Reply by linux-lukas** [reply]
Mastodon is a decentrally organised alternative Twitter and part of Fediverse. 

- Mastodon is without advertising and tracking
- Mastodon is decentrally organised
- Mastodon is part of the Fediverse
- Mastodon is free and open source.

Sharing project information and project announcements would be possible via a Mastodon account. Interaction with the Fediverse community would also be possible.

**Reply by joseantmazonsb** [reply]
mm maybe I'll consider it when the first stable version is ready, but I don't see the point right now
